client/extensions/woocommerce/woocommerce-services/lib/initialize-labels-state/index.js

```diff
@@ -106,6 +106,9 @@ export default data => {
 				normalizationInProgress: false,
 				allowChangeCountry: Boolean( canChangeCountries ),
 			},
+			endShipper: {
+				values: formData.end_shipper,
```

Comment:
I noticed the other fields use other keys than just `values`. Do you know how this works?

Reply:
I see : `isNormalized`, `normalized`, `ignoreValidation`, `selectNormalized`, `normalizationInProgress`, `allowChangeCountry` keys on the other object. 
And from what I see, these keys : `ignoreValidation`, `selectNormalized`, `normalizationInProgress`, `allowChangeCountry` are meant for flags in the address UI. And since end shipper doesn't have UI at the moment, IMO we won't need to use that.

`normalized` is address info that has been normalized and saved into database (e.g. normalized origin address is saved on `wp_options` under `wc_connect_origin_address` option name ).
`isNormalized` is used for a flag whether the current address is used normalized address or not.

I did some simple tests when sending an endshipper address like this : 
```
street1: 60 29TH ST,
street2: # 343,
city: SAN FRANCISCO,
state: CA,
zip:94110,
country: US
```
endShipper API seems responding with normalized address like below : 
```
street1: 60 29TH ST # 343,
street2: null,
city: SAN FRANCISCO,
state: CA,
zip:94110-4929,
country: US
```

Wondering if we need to save this normalized address in `wp_options` like origin address do? Or should we just leave it as we don't save the end shipper address in the WCC Server DB?